packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +310,57 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// An picker default magnifier, grey rrect first introduced in iOS 14
+class CupertinoPickerDefaultMagnifier extends StatelessWidget {
+
+  /// Create a magnifier, it can be easily configured with the default style.
+  /// The default background is [CupertinoColors.tertiarySystemFill],
+  /// which also has default radius and margin on the left and right sides.
+  const CupertinoPickerDefaultMagnifier({
+    Key key,
+    this.background = CupertinoColors.tertiarySystemFill,
+    this.useLeftStyle = true,
+    this.useRightStyle = true,
+  }) : assert(background != null),
+       assert(useLeftStyle != null),
+       assert(useRightStyle != null),
+       super(key: key);
+
+  /// Whether to use the default radius and margin on the left side
+  final bool useLeftStyle;
+
+  /// Whether to use the default radius and margin on the right side
+  final bool useRightStyle;
+
+  /// The color to fill in the background of the magnifier, Support for using [CupertinoDynamicColor].
+  final Color background;
+
+  /// default margin of the 'magnifier'.
```

Comment:
Use complete sentences for all documentations (e.g. capitalize first letter).

Reply:
That was my mistake and it's now resolved.